src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  line-height: normal;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+:root {
+  --main-color: black;
+  --active-color: #00acdc;
+}
+
+.page {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
```

Comment:
The `.nav__item` class uses `display: flex;`, which might not be necessary if each navigation link is wrapped in its own `<li>` element. Consider reviewing the HTML structure to ensure semantic correctness and potentially simplify the CSS.